freqtrade/plot/plotting.py

```diff
@@ -48,11 +49,21 @@ def init_plotscript(config):
         data_format=config.get('dataformat_ohlcv', 'json'),
     )
 
-    trades = load_trades(config['trade_source'],
-                         db_url=config.get('db_url'),
-                         exportfilename=config.get('exportfilename'),
-                         )
+    skip_trades = False
+    if not isfile(config.get('exportfilename')) and config['trade_source'] == 'file':
+        logger.warning("Backtest file is missing skipping trades.")
+        skip_trades = True
+    elif config.get('skip_trades', False):
+        skip_trades = True
```

Comment:
I think you'll need to revert the sequence of the if statements, and check the explicit `skip_trades` parameter first.

If i call this with `--export-filename nonexistingfile.json --skip-trades` - it's printing "Backtestfile is missing ..." - but it should not (same without --export-filename if the file does not exist - but i'm too lazy to delete the file).

Reply:
fixed in next commit